.github/workflows/generate-telecon-agenda.yml

```diff
@@ -0,0 +1,38 @@
+name: Generate Telecon Agenda
+on:
+  # We can manually dispatch this workflow to re-build the agenda at any time.
+  workflow_dispatch:
+  schedule:
+    # ... but it also automatically runs every Wednesday at midnight UTC
+    - cron: "0 0 * * 3"
```

Comment:
Perhaps run it at 5 minutes after midnight, just to avoid weird race conditions with the `date` call below? It should work as-is, but putting it right at midnight makes me nervous.

Reply:
I picked 8:30am UTC instead which should not be midnight in any timezone